navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/NavigationService.java

```diff
@@ -193,14 +186,6 @@ public void endNavigation() {
     // Remove the this navigation service progress change listener
     progressChangeListeners.remove(this);
 
-    responseHandler.removeCallbacksAndMessages(null);
-
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2) {
-      locationUpdatedThread.quitSafely();
-    } else {
-      locationUpdatedThread.quit();
-    }
-
     // Lower accuracy to minimize battery usage while not in navigation mode.
     // TODO restore accuracy state to what user had before nav session
```

Comment:
ticketed in https://github.com/mapbox/mapbox-navigation-android/issues/38 and removed.